src/tasks/sep/039-U17-sep-mmz/.env

```diff
@@ -0,0 +1 @@
+TENDERLY_GAS=30000000
```

Comment:
This task uses 15,201,063 gas, so it fits into the fusaka gas limit, and we do not need to split this one up